testutil/cmd.go

```diff
@@ -0,0 +1,22 @@
+package testutil
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// SetArgs sets arguments for the command. It is desired to replace the cmd.SetArgs in all test case, as cmd.SetArgs doesn't reset flag value as expected.
+//
+// see https://github.com/spf13/cobra/issues/2079#issuecomment-1867991505 for more detail info
+func SetArgs(cmd *cobra.Command, args []string) {
```

Comment:
We should be mindful of new public api we add. Can we maybe have this in an internal package?

Concerning using it everywhere in the sdk, we should really really watch out that we don't increase the reliance on the sdk for already extracted modules.

Reply:
@julienrbrt Hi sir, I have move it to `internal/testutil`